docs/guide/tutorials.rst

```diff
@@ -9,134 +9,199 @@ Structure of Data Sets
 SmartPeak is optimised to process data set folders with the following structure :
 
 #. featureFilterComponentGroups.csv
-	This file contains component group names and their properties such as ``retention_time`` and ``intensity`` with the
-	header in the following format :
-	``component_group_name,n_heavy_l ,n_heavy_u,n_light_l,n_light_u,n_detecting_l,``
-	``n_detecting_u,n_quantifying_l,n_quantifying_u,n_identifying_l,n_identifying_u,``
-	``n_transitions_l,n_transitions_u,ion_ratio_pair_name_1,ion_ratio_pair_name_2,``
-	``ion_ratio_l,ion_ratio_u,retention_time_l,retention_time_u,intensity_l,``
-	``intensity_u,overall_quality_l,overall_quality_u,retention_time_l_,``
-	``retention_time_u_``.
-	A sample row may look like the following :
-	``Serotonin,0,10,0,10,0,10,0,10,0,10,0,10,,,,,1.37,2.37,``
-	``-1.00E+12,1.00E+15,-1.00E+12,1.00E+12,82.2,142.2``.
+	This file contains component group names and their properties such as ``retention_time`` and ``intensity``. 
+	The header including a sample entry is shown below:
+
+	.. table:: featureFilterComponentGroups.csv Headers
+		:widths: auto
+
+		==================== ========= ========= ========= ========= ============= ============= =============== =============== =============== =============== =============== =============== ===================== ===================== =========== =========== ================ ================ =========== =========== ================= ================= ================= =================
+		component_group_name n_heavy_l n_heavy_u n_light_l n_light_u n_detecting_l n_detecting_u n_quantifying_l n_quantifying_u n_identifying_l n_identifying_u n_transitions_l n_transitions_u ion_ratio_pair_name_1 ion_ratio_pair_name_2 ion_ratio_l ion_ratio_u retention_time_l retention_time_u intensity_l intensity_u overall_quality_l overall_quality_u retention_time_l_ retention_time_u_
```

Comment:
Please remove the last two columns retention_time_l_ and retention_time_u_.  They are bogus columns.